stan/math/prim/fun/acos.hpp

```diff
@@ -9,7 +9,7 @@ namespace stan {
 namespace math {
 
 /**
- * Structure to wrap acos() so it can be vectorized.
+ * Structure to wrap `acos()` so it can be vectorized.
```

Comment:
Thanks! Didn't know we could use markdown until Bob showed me. Much easier!

Reply:
Yeah, me neither. @mcol let me in on the secret :)